var/spack/repos/builtin/packages/spykfunc/package.py

```diff
@@ -23,10 +23,11 @@ class Spykfunc(PythonPackage):
     version('0.15.1', tag='v0.15.1', submodules=True, get_full_repo=True)
     version('0.15.2', tag='v0.15.2', submodules=True, get_full_repo=True)
     version('0.15.3', tag='v0.15.3', submodules=True, get_full_repo=True)
+    version('0.15.4', tag='v0.15.4', submodules=True, get_full_repo=True)
```

Comment:
Why are the versions in reverse order...?

Reply:
"historical reasons" :)